src/input/DBusInputDriver.cc

```diff
@@ -150,3 +150,10 @@ const std::string & DBusInputDriver::get_name() const
 {
     return name;
 }
+
+std::string DBusInputDriver::get_info() const
+{
+	std::ostringstream stream;
+	stream << get_name() << " " << (isOpen() ? "open" : "not open");
+	return stream.str();
```

Comment:
You can replace this by `return STR(get_name() << " " << (isOpen() ? "open" : "not open"))`

-> We have a STR macro for this common operation: https://github.com/openscad/openscad/blob/master/src/printutils.h#L100